main.js

```diff
@@ -16,13 +16,44 @@ var titleInput = document.getElementById('title-input');
 var bodyInput = document.getElementById('body-input');
 var cardGrid = document.getElementById('card-grid');
 var inputFields = document.querySelectorAll('textarea');
+var starBorder = document.getElementById('star-border')
+var ideaBoxes = document.querySelector(".idea-boxes")
 
 //Event Listeners
 titleInput.addEventListener('keydown', buttonDisable);
 bodyInput.addEventListener('keydown', buttonDisable);
 saveButton.addEventListener('click', saveNewIdea);
+// deleteButton.addEventListener('click', deleteIdeaCard)
+
+cardGrid.addEventListener('click', function(event){
+
+  if(event.target.classList.contains('delete')) {
```

Comment:
Love it! Glad we got this working! We were right there. Were grabbing the wrong parent

Reply:
Great job Michele, applying our lessons this week!